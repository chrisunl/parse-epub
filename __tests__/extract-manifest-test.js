import { contentOpf } from './data.js'
import extractManifest from '../extract/manifest.js'

test('#extractManifest', () => {
  expect(extractManifest(contentOpf)).toMatchSnapshot();
});
