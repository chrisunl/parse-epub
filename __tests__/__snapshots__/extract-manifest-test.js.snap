exports[`test #extractManifest 1`] = `
Object {
  "byId": Object {
    "item-1377": Object {
      "href": "s9ml/glossary.xhtml",
      "id": "item-1377",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-1471": Object {
      "href": "toc.xhtml",
      "id": "item-1471",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "nav",
    },
    "item-data-uuid-01eee42463c54f1b945502ebf51ab9ab": Object {
      "href": "s9ml/culture_and_belonging/what_is_a_culture.xhtml",
      "id": "item-data-uuid-01eee42463c54f1b945502ebf51ab9ab",
      "mediaOverlay": "s0038",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-0cf7f56918514bb784e1de2b5e739da1": Object {
      "href": "s9ml/the_thrill_of_horror/scary_history_100_years_of_monster_movies.xhtml",
      "id": "item-data-uuid-0cf7f56918514bb784e1de2b5e739da1",
      "mediaOverlay": "s0059",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-0f371090c0e84cd09a06f868c3fe8efd": Object {
      "href": "s9ml/approaching_adulthood/the_birth_of_the_teenager.xhtml",
      "id": "item-data-uuid-0f371090c0e84cd09a06f868c3fe8efd",
      "mediaOverlay": "s0015",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-0f7b7bf0ed1b4c5c88b86bf81c8e2f92": Object {
      "href": "s9ml/approaching_adulthood/approaching_adulthood_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-0f7b7bf0ed1b4c5c88b86bf81c8e2f92",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-101710cd602a4fcfa90108c5c0347df7": Object {
      "href": "s9ml/anne_franks_legacy/anne_frank_s_legacy_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-101710cd602a4fcfa90108c5c0347df7",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-107c288bb253478f8a3094fdd57e00d0": Object {
      "href": "s9ml/the_thrill_of_horror/the_open_window.xhtml",
      "id": "item-data-uuid-107c288bb253478f8a3094fdd57e00d0",
      "mediaOverlay": "s0061",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-14c6d82a58e64980992078f1726dac96": Object {
      "href": "s9ml/the_move_toward_freedom/from_reminiscences_of_my_life_in_camp.xhtml",
      "id": "item-data-uuid-14c6d82a58e64980992078f1726dac96",
      "mediaOverlay": "s0051",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-1695c0869dbf4fc38e255f1d8571b4e5": Object {
      "href": "s9ml/the_value_of_work/weird_jobs_around_the_world.xhtml",
      "id": "item-data-uuid-1695c0869dbf4fc38e255f1d8571b4e5",
      "mediaOverlay": "s0071",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-17a19334765e441d848dd98a07d3b583": Object {
      "href": "s9ml/approaching_adulthood/approaching_adulthood_opener.xhtml",
      "id": "item-data-uuid-17a19334765e441d848dd98a07d3b583",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-3a07ddda3070483e99e2b20349853c76": Object {
      "href": "s9ml/the_value_of_work/dawn_to_dusk_a_child_s_workday.xhtml",
      "id": "item-data-uuid-3a07ddda3070483e99e2b20349853c76",
      "mediaOverlay": "s0064",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-3a5daf183fc34c31822a2b16d13a57ba": Object {
      "href": "s9ml/anne_franks_legacy/from_anne_frank_remembered.xhtml",
      "id": "item-data-uuid-3a5daf183fc34c31822a2b16d13a57ba",
      "mediaOverlay": "s0005",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-3b13d68a9f9447fe8bd6f2d7c7ea7403": Object {
      "href": "s9ml/overview/table_of_contents.xhtml",
      "id": "item-data-uuid-3b13d68a9f9447fe8bd6f2d7c7ea7403",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-3c0bb8315ea14a2bac12ace68240caff": Object {
      "href": "s9ml/the_thrill_of_horror/the_thrill_of_horror_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-3c0bb8315ea14a2bac12ace68240caff",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-40c45e21a58c438283bf95dcaa809537": Object {
      "href": "s9ml/culture_and_belonging/family_reunion.xhtml",
      "id": "item-data-uuid-40c45e21a58c438283bf95dcaa809537",
      "mediaOverlay": "s0033",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-445da3f24e8d49d897c41c17a615bcef": Object {
      "href": "s9ml/anne_franks_legacy/a_star_who_s_anne_s_age.xhtml",
      "id": "item-data-uuid-445da3f24e8d49d897c41c17a615bcef",
      "mediaOverlay": "s0001",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-47a9d3e21464491cb37d718162b9b520": Object {
      "href": "s9ml/the_thrill_of_horror/the_horror.xhtml",
      "id": "item-data-uuid-47a9d3e21464491cb37d718162b9b520",
      "mediaOverlay": "s0060",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-553be8a225384235b06f21698e403a16": Object {
      "href": "s9ml/approaching_adulthood/jack_andraka_teen_scientist.xhtml",
      "id": "item-data-uuid-553be8a225384235b06f21698e403a16",
      "mediaOverlay": "s0012",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-589d9984380e4568b1eb5710e6e81ca2": Object {
      "href": "s9ml/the_value_of_work/harnessing_the_wind.xhtml",
      "id": "item-data-uuid-589d9984380e4568b1eb5710e6e81ca2",
      "mediaOverlay": "s0065",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-5adbebda98604da48e98180467c64cfd": Object {
      "href": "s9ml/the_thrill_of_horror/the_thrill_of_horror_opener.xhtml",
      "id": "item-data-uuid-5adbebda98604da48e98180467c64cfd",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-5b23866eff924613af22d8a296c61653": Object {
      "href": "s9ml/culture_and_belonging/selena_s_blog.xhtml",
      "id": "item-data-uuid-5b23866eff924613af22d8a296c61653",
      "mediaOverlay": "s0036",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-601228182dbc40e2ba71787be4c05654": Object {
      "href": "s9ml/approaching_adulthood/old_enough_to_vote.xhtml",
      "id": "item-data-uuid-601228182dbc40e2ba71787be4c05654",
      "mediaOverlay": "s0014",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-61380725c0dd484593df2807e84cf853": Object {
      "href": "s9ml/anne_franks_legacy/the_hitler_youth.xhtml",
      "id": "item-data-uuid-61380725c0dd484593df2807e84cf853",
      "mediaOverlay": "s0007",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-6ad2f32297f84d1da04638f7e582c374": Object {
      "href": "s9ml/culture_and_belonging/culture_and_belonging_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-6ad2f32297f84d1da04638f7e582c374",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-6d74e8e547ec455a9c00ebb37cb4a872": Object {
      "href": "s9ml/culture_and_belonging/immigrants.xhtml",
      "id": "item-data-uuid-6d74e8e547ec455a9c00ebb37cb4a872",
      "mediaOverlay": "s0034",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-74687195b0fb413bba6b9ca5d8aa3cec": Object {
      "href": "s9ml/anne_franks_legacy/dear_anne.xhtml",
      "id": "item-data-uuid-74687195b0fb413bba6b9ca5d8aa3cec",
      "mediaOverlay": "s0004",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-757f293b2be647deaa1c5291da00413e": Object {
      "href": "s9ml/anne_franks_legacy/seven_reasons_to_keep_a_diary.xhtml",
      "id": "item-data-uuid-757f293b2be647deaa1c5291da00413e",
      "mediaOverlay": "s0006",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-7841571fa25b470fa9b4158ce0544f87": Object {
      "href": "s9ml/the_value_of_work/take_a_break_and_other_ways_to_get_more_work_done.xhtml",
      "id": "item-data-uuid-7841571fa25b470fa9b4158ce0544f87",
      "mediaOverlay": "s0067",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-7bf20da5723c4a5799c86f97c2fd81b3": Object {
      "href": "s9ml/the_thrill_of_horror/bringing_zombies_to_life.xhtml",
      "id": "item-data-uuid-7bf20da5723c4a5799c86f97c2fd81b3",
      "mediaOverlay": "s0056",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-8fc059832a1f4c64be19490536ebc955": Object {
      "href": "s9ml/the_value_of_work/the_value_of_work_opener.xhtml",
      "id": "item-data-uuid-8fc059832a1f4c64be19490536ebc955",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-999b44bf3f03460cbdf41781bd702168": Object {
      "href": "s9ml/culture_and_belonging/immigration_and_migration_in_the_united_states.xhtml",
      "id": "item-data-uuid-999b44bf3f03460cbdf41781bd702168",
      "mediaOverlay": "s0035",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-9a306d7babff42c084fa63ad29079c97": Object {
      "href": "s9ml/the_thrill_of_horror/its_a_mist_its_a_bat_its_a_._._._vampire.xhtml",
      "id": "item-data-uuid-9a306d7babff42c084fa63ad29079c97",
      "mediaOverlay": "s0058",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb": Object {
      "href": "s9ml/the_move_toward_freedom/the_move_toward_freedom_opener.xhtml",
      "id": "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f": Object {
      "href": "s9ml/anne_franks_legacy/anne_frank_s_legacy_opener.xhtml",
      "id": "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-bda8bcd0fdaa4c26be8914772d6c84ec": Object {
      "href": "s9ml/the_value_of_work/the_conduit_mouse.xhtml",
      "id": "item-data-uuid-bda8bcd0fdaa4c26be8914772d6c84ec",
      "mediaOverlay": "s0068",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-cc1ee917649f4f55855784812d45ef9d": Object {
      "href": "s9ml/the_value_of_work/i_hear_america_singing.xhtml",
      "id": "item-data-uuid-cc1ee917649f4f55855784812d45ef9d",
      "mediaOverlay": "s0066",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-cc3d8d64c26046bc92151bfe70170c4c": Object {
      "href": "s9ml/approaching_adulthood/celeste_s_heart.xhtml",
      "id": "item-data-uuid-cc3d8d64c26046bc92151bfe70170c4c",
      "mediaOverlay": "s0011",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d11317bd1e344490a568604ca1f4616f": Object {
      "href": "s9ml/approaching_adulthood/what_teenagers_need_to_thrive.xhtml",
      "id": "item-data-uuid-d11317bd1e344490a568604ca1f4616f",
      "mediaOverlay": "s0016",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d15b805965d14521aeb48d93d168b65f": Object {
      "href": "s9ml/the_thrill_of_horror/eeeek_how_to_scare_your_friends.xhtml",
      "id": "item-data-uuid-d15b805965d14521aeb48d93d168b65f",
      "mediaOverlay": "s0057",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d17e58059b0542f1a831de1593127652": Object {
      "href": "s9ml/culture_and_belonging/welcome_to_california.xhtml",
      "id": "item-data-uuid-d17e58059b0542f1a831de1593127652",
      "mediaOverlay": "s0037",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d1dc38f40de742b5a24353c05ed1a23e": Object {
      "href": "s9ml/the_move_toward_freedom/the_freedom_train_keeps_rolling.xhtml",
      "id": "item-data-uuid-d1dc38f40de742b5a24353c05ed1a23e",
      "mediaOverlay": "s0053",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc": Object {
      "href": "s9ml/culture_and_belonging/culture_and_belonging_opener.xhtml",
      "id": "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-d43a2540b0774b5f973c841873a8b814": Object {
      "href": "s9ml/the_move_toward_freedom/from_the_red_badge_of_courage.xhtml",
      "id": "item-data-uuid-d43a2540b0774b5f973c841873a8b814",
      "mediaOverlay": "s0052",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-debfcf17707946f0bf220b7924adaa48": Object {
      "href": "s9ml/the_move_toward_freedom/freedom_spreads_through_the_states.xhtml",
      "id": "item-data-uuid-debfcf17707946f0bf220b7924adaa48",
      "mediaOverlay": "s0050",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-e8573f88150047078fdb5bfb083ca5c7": Object {
      "href": "s9ml/approaching_adulthood/moving_forward_looking_back.xhtml",
      "id": "item-data-uuid-e8573f88150047078fdb5bfb083ca5c7",
      "mediaOverlay": "s0013",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-ea0242f6d23d4e608c7252ad571882f5": Object {
      "href": "s9ml/the_move_toward_freedom/the_move_toward_freedom_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-ea0242f6d23d4e608c7252ad571882f5",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-ec172a4c778d409da1aec2f5474cc337": Object {
      "href": "s9ml/the_value_of_work/the_value_of_work_copyright_acknowledgments_and_image_credits.xhtml",
      "id": "item-data-uuid-ec172a4c778d409da1aec2f5474cc337",
      "mediaOverlay": undefined,
      "mediaType": "application/xhtml+xml",
      "properties": undefined,
    },
    "item-data-uuid-f7923d60c5894cc999346a9f2c8f5ae7": Object {
      "href": "s9ml/anne_franks_legacy/the_legacy_lives_on.xhtml",
      "id": "item-data-uuid-f7923d60c5894cc999346a9f2c8f5ae7",
      "mediaOverlay": "s0008",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-fb124defa1ef4c04a4f79397ae047df0": Object {
      "href": "s9ml/the_move_toward_freedom/frederick_douglass.xhtml",
      "id": "item-data-uuid-fb124defa1ef4c04a4f79397ae047df0",
      "mediaOverlay": "s0049",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "item-data-uuid-ffc2e9c0ab5a404980a1c2686659ab6f": Object {
      "href": "s9ml/the_move_toward_freedom/a_final_escape.xhtml",
      "id": "item-data-uuid-ffc2e9c0ab5a404980a1c2686659ab6f",
      "mediaOverlay": "s0048",
      "mediaType": "application/xhtml+xml",
      "properties": "svg",
    },
    "s0001": Object {
      "href": "smil/anne_franks_legacy/a_star_who_s_anne_s_age.xhtml.smil",
      "id": "s0001",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0004": Object {
      "href": "smil/anne_franks_legacy/dear_anne.xhtml.smil",
      "id": "s0004",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0005": Object {
      "href": "smil/anne_franks_legacy/from_anne_frank_remembered.xhtml.smil",
      "id": "s0005",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0006": Object {
      "href": "smil/anne_franks_legacy/seven_reasons_to_keep_a_diary.xhtml.smil",
      "id": "s0006",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0007": Object {
      "href": "smil/anne_franks_legacy/the_hitler_youth.xhtml.smil",
      "id": "s0007",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0008": Object {
      "href": "smil/anne_franks_legacy/the_legacy_lives_on.xhtml.smil",
      "id": "s0008",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0011": Object {
      "href": "smil/approaching_adulthood/celeste_s_heart.xhtml.smil",
      "id": "s0011",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0012": Object {
      "href": "smil/approaching_adulthood/jack_andraka_teen_scientist.xhtml.smil",
      "id": "s0012",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0013": Object {
      "href": "smil/approaching_adulthood/moving_forward_looking_back.xhtml.smil",
      "id": "s0013",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0014": Object {
      "href": "smil/approaching_adulthood/old_enough_to_vote.xhtml.smil",
      "id": "s0014",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0015": Object {
      "href": "smil/approaching_adulthood/the_birth_of_the_teenager.xhtml.smil",
      "id": "s0015",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0016": Object {
      "href": "smil/approaching_adulthood/what_teenagers_need_to_thrive.xhtml.smil",
      "id": "s0016",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0033": Object {
      "href": "smil/culture_and_belonging/family_reunion.xhtml.smil",
      "id": "s0033",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0034": Object {
      "href": "smil/culture_and_belonging/immigrants.xhtml.smil",
      "id": "s0034",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0035": Object {
      "href": "smil/culture_and_belonging/immigration_and_migration_in_the_united_states.xhtml.smil",
      "id": "s0035",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0036": Object {
      "href": "smil/culture_and_belonging/selena_s_blog.xhtml.smil",
      "id": "s0036",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0037": Object {
      "href": "smil/culture_and_belonging/welcome_to_california.xhtml.smil",
      "id": "s0037",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0038": Object {
      "href": "smil/culture_and_belonging/what_is_a_culture.xhtml.smil",
      "id": "s0038",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0048": Object {
      "href": "smil/the_move_toward_freedom/a_final_escape.xhtml.smil",
      "id": "s0048",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0049": Object {
      "href": "smil/the_move_toward_freedom/frederick_douglass.xhtml.smil",
      "id": "s0049",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0050": Object {
      "href": "smil/the_move_toward_freedom/freedom_spreads_through_the_states.xhtml.smil",
      "id": "s0050",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0051": Object {
      "href": "smil/the_move_toward_freedom/from_reminiscences_of_my_life_in_camp.xhtml.smil",
      "id": "s0051",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0052": Object {
      "href": "smil/the_move_toward_freedom/from_the_red_badge_of_courage.xhtml.smil",
      "id": "s0052",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0053": Object {
      "href": "smil/the_move_toward_freedom/the_freedom_train_keeps_rolling.xhtml.smil",
      "id": "s0053",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0056": Object {
      "href": "smil/the_thrill_of_horror/bringing_zombies_to_life.xhtml.smil",
      "id": "s0056",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0057": Object {
      "href": "smil/the_thrill_of_horror/eeeek_how_to_scare_your_friends.xhtml.smil",
      "id": "s0057",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0058": Object {
      "href": "smil/the_thrill_of_horror/its_a_mist_its_a_bat_its_a_._._._vampire.xhtml.smil",
      "id": "s0058",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0059": Object {
      "href": "smil/the_thrill_of_horror/scary_history_100_years_of_monster_movies.xhtml.smil",
      "id": "s0059",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0060": Object {
      "href": "smil/the_thrill_of_horror/the_horror.xhtml.smil",
      "id": "s0060",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0061": Object {
      "href": "smil/the_thrill_of_horror/the_open_window.xhtml.smil",
      "id": "s0061",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0064": Object {
      "href": "smil/the_value_of_work/dawn_to_dusk_a_child_s_workday.xhtml.smil",
      "id": "s0064",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0065": Object {
      "href": "smil/the_value_of_work/harnessing_the_wind.xhtml.smil",
      "id": "s0065",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0066": Object {
      "href": "smil/the_value_of_work/i_hear_america_singing.xhtml.smil",
      "id": "s0066",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0067": Object {
      "href": "smil/the_value_of_work/take_a_break_and_other_ways_to_get_more_work_done.xhtml.smil",
      "id": "s0067",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0068": Object {
      "href": "smil/the_value_of_work/the_conduit_mouse.xhtml.smil",
      "id": "s0068",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
    "s0071": Object {
      "href": "smil/the_value_of_work/weird_jobs_around_the_world.xhtml.smil",
      "id": "s0071",
      "mediaOverlay": undefined,
      "mediaType": "application/smil+xml",
      "properties": undefined,
    },
  },
  "items": Array [
    "item-data-uuid-3b13d68a9f9447fe8bd6f2d7c7ea7403",
    "item-data-uuid-d2a6e8f4db704bae875c9b4c5dc209cc",
    "item-data-uuid-6d74e8e547ec455a9c00ebb37cb4a872",
    "item-data-uuid-5b23866eff924613af22d8a296c61653",
    "item-data-uuid-40c45e21a58c438283bf95dcaa809537",
    "item-data-uuid-999b44bf3f03460cbdf41781bd702168",
    "item-data-uuid-01eee42463c54f1b945502ebf51ab9ab",
    "item-data-uuid-d17e58059b0542f1a831de1593127652",
    "item-data-uuid-6ad2f32297f84d1da04638f7e582c374",
    "item-data-uuid-5adbebda98604da48e98180467c64cfd",
    "item-data-uuid-9a306d7babff42c084fa63ad29079c97",
    "item-data-uuid-7bf20da5723c4a5799c86f97c2fd81b3",
    "item-data-uuid-47a9d3e21464491cb37d718162b9b520",
    "item-data-uuid-107c288bb253478f8a3094fdd57e00d0",
    "item-data-uuid-d15b805965d14521aeb48d93d168b65f",
    "item-data-uuid-0cf7f56918514bb784e1de2b5e739da1",
    "item-data-uuid-3c0bb8315ea14a2bac12ace68240caff",
    "item-data-uuid-a4f73db0c4ff4464afdbdef6540a1feb",
    "item-data-uuid-ffc2e9c0ab5a404980a1c2686659ab6f",
    "item-data-uuid-fb124defa1ef4c04a4f79397ae047df0",
    "item-data-uuid-d43a2540b0774b5f973c841873a8b814",
    "item-data-uuid-14c6d82a58e64980992078f1726dac96",
    "item-data-uuid-debfcf17707946f0bf220b7924adaa48",
    "item-data-uuid-d1dc38f40de742b5a24353c05ed1a23e",
    "item-data-uuid-ea0242f6d23d4e608c7252ad571882f5",
    "item-data-uuid-17a19334765e441d848dd98a07d3b583",
    "item-data-uuid-0f371090c0e84cd09a06f868c3fe8efd",
    "item-data-uuid-553be8a225384235b06f21698e403a16",
    "item-data-uuid-cc3d8d64c26046bc92151bfe70170c4c",
    "item-data-uuid-e8573f88150047078fdb5bfb083ca5c7",
    "item-data-uuid-601228182dbc40e2ba71787be4c05654",
    "item-data-uuid-d11317bd1e344490a568604ca1f4616f",
    "item-data-uuid-0f7b7bf0ed1b4c5c88b86bf81c8e2f92",
    "item-data-uuid-ad4d37cb6ecb4c11ada0eafc3c44146f",
    "item-data-uuid-3a5daf183fc34c31822a2b16d13a57ba",
    "item-data-uuid-61380725c0dd484593df2807e84cf853",
    "item-data-uuid-445da3f24e8d49d897c41c17a615bcef",
    "item-data-uuid-74687195b0fb413bba6b9ca5d8aa3cec",
    "item-data-uuid-757f293b2be647deaa1c5291da00413e",
    "item-data-uuid-f7923d60c5894cc999346a9f2c8f5ae7",
    "item-data-uuid-101710cd602a4fcfa90108c5c0347df7",
    "item-data-uuid-8fc059832a1f4c64be19490536ebc955",
    "item-data-uuid-cc1ee917649f4f55855784812d45ef9d",
    "item-data-uuid-3a07ddda3070483e99e2b20349853c76",
    "item-data-uuid-bda8bcd0fdaa4c26be8914772d6c84ec",
    "item-data-uuid-589d9984380e4568b1eb5710e6e81ca2",
    "item-data-uuid-1695c0869dbf4fc38e255f1d8571b4e5",
    "item-data-uuid-7841571fa25b470fa9b4158ce0544f87",
    "item-data-uuid-ec172a4c778d409da1aec2f5474cc337",
    "item-1377",
    "item-1471",
    "s0071",
    "s0068",
    "s0067",
    "s0066",
    "s0065",
    "s0064",
    "s0061",
    "s0060",
    "s0059",
    "s0058",
    "s0057",
    "s0056",
    "s0053",
    "s0052",
    "s0051",
    "s0050",
    "s0049",
    "s0048",
    "s0038",
    "s0037",
    "s0036",
    "s0035",
    "s0034",
    "s0033",
    "s0016",
    "s0015",
    "s0014",
    "s0013",
    "s0012",
    "s0011",
    "s0008",
    "s0007",
    "s0006",
    "s0005",
    "s0004",
    "s0001",
  ],
}
`;
